Integrations/LogRhythmRest/LogRhythmRest.py

```diff
@@ -0,0 +1,279 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+
+import json
+import requests
+import random
+import string
+from datetime import datetime, timedelta
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+''' GLOBALS/PARAMS '''
+
+TOKEN = demisto.params().get('token')
+BASE_URL = demisto.params()['url'][:-1] if (demisto.params()['url'] and demisto.params()['url'].endswith('/')) \
```

Comment:
make sure to remove the rest of the command in the next line
```suggestion
BASE_URL = demisto.params()['url'].strip('/')
```